### ISSUE TITLE
Current plugin release not compatible with newest Intellij release

### ISSUE THREAD

**Issue by rrjjvv** [earthly/earthly-intellij-plugin#10]
After upgrading my IDE (Intellij Ultimate), the Earthly plugin fails to load with error
```
Plugin 'Earthly' (version '0.0.3-ps') is not compatible with the current version of the IDE, because it requires build 223.* or older but the current build is IU-231.8109.175
```

Full IDE details:
```
Build #IU-231.8109.175, built on March 28, 2023

Licensed to <me>
Subscription is active until February 8, 2024.

Runtime version: 17.0.6+10-b829.5 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
```

**Comment by idelvall** [comment]
cc @brandonSc

**Comment by genestack-okunitsyn** [comment]
Facing this issue too

**Comment by rrjjvv** [comment]
@genestack-okunitsyn in the interim, you can build the plugin from `main` (it appears to be fixed, just unreleased).  I had made nearly the same change in my fork and have been using the result for a few days now.

If I had to guess, they might be waiting for https://github.com/earthly/earthly/pull/2836 before releasing a new version (which would give us syntax highlighting for some of the newer keywords).

**Comment by idelvall** [comment]
Hi, yes it has been fixed and released, but not reviewed yet by Jetbrains.
Tomorrow it should be ready
https://plugins.jetbrains.com/plugin/20392-earthly/versions

**Comment by idelvall** [comment]
Fixed in `0.0.4-x`  https://plugins.jetbrains.com/plugin/20392-earthly/versions
